Dockerfile.graphql-api

```diff
@@ -0,0 +1,20 @@
+FROM python:3.6.12-buster AS build
```

Comment:
Any particular reason you went with python 3.6.12 and not a later version?

Reply:
3.6.12 is the latest in the 3.6 series. I went with 3.6 rather than 3.7 or 3.8 since that's the version listed in tox.ini. If we want to use a more recent Python version I'm totally on board, but it should be a separate commit.

Reply:
Agree with doing this in another commit. I tried upgrading once and there were a few problems with dependencies.

We should strive for using the latest python version